### ISSUE TITLE
Share API

### ISSUE THREAD

**Issue by kolumb** [sdegueldre/pong-3.0#14]
Copy link button besides copying url could also make call to share API that works on some mobile browsers.
https://css-tricks.com/how-to-use-the-web-share-api/
https://caniuse.com/web-share

**Comment by sdegueldre** [comment]
True, since I didn't really think about mobile support before your PR I never really thought about this but it could be really cool!
